pkg/build/api/validation/validation_test.go

```diff
@@ -67,7 +63,7 @@ func TestBuildValidationFailure(t *testing.T) {
 			Phase: buildapi.BuildPhaseNew,
 		},
 	}
-	if result := ValidateBuild(build); len(result) != 3 {
+	if result := ValidateBuild(build); len(result) != 2 {
```

Comment:
This looks like a bad test. Why just count the number of errors instead of actually checking what errors were returned?